pages/common/ssh.md

```diff
@@ -31,6 +27,10 @@
 
 `ssh -J {{username}}@{{jump_host}} {{username}}@{{remote_host}}`
 
+- SSH reverse proxy: Expose a localhost port on the remote server (Set `GatewayPorts yes` in sshd_config to allow internet access):
```

Comment:
Two things here:
1. Does "localhost port on the remote server" mean a port that is open on the server, localhost referring to the server itself?
2. "Set `GatewayPorts yes` in sshd_config" &mdash; is `sshd_config` on the server or client?

Also, will you please put `sshd_config` in backticks per WIP PR #5128?